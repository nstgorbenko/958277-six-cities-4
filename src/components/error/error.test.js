import React from "react";
import renderer from "react-test-renderer";

import Error from "./error.jsx";

describe(`Error Component rendering`, () => {
  it(`Error Component should render correctly`, () => {
    const tree = renderer
      .create(<Error />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
