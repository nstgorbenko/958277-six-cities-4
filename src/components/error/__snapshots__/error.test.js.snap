// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component rendering Error Component should render correctly 1`] = `
<div
  className="page page--gray page--main"
>
  <main
    className="page__main page__main--index page__main--index-empty"
  >
    <div
      className="cities"
    >
      <div
        className="cities__places-container cities__places-container--empty container"
      >
        <section
          className="cities__no-places"
        >
          <div
            className="cities__status-wrapper tabs__content"
          >
            <b
              className="cities__status"
            >
              Oops!
            </b>
            <p
              className="cities__status-description"
            >
              Something went wrong.
            </p>
            <p
              className="cities__status-description"
            >
              Please refresh page to load data.
            </p>
          </div>
        </section>
        <div
          className="cities__right-section"
        />
      </div>
    </div>
  </main>
</div>
`;
